testnets/mantrachaintestnet/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mantrachaintestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "MANTRA Chain",
+  "chain_id": "mantrachain-testnet-1",
+  "bech32_prefix": "uaum",
+  "daemon_name": "mantrachaind",
+  "node_home": "$HOME/.mantrachain",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uaum",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.001,
+        "average_gas_price": 0.002,
+        "high_gas_price": 0.003
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uaum"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/MANTRA-Finance",
+    "recommended_version": "v1.0.0",
```

Comment:
please remove the 'v' here, or add 'v' to the compatible_versions (either way, it should be consistent)

Reply:
Removed 'v'.